
From: Nick Piggin <nickpiggin@yahoo.com.au>

From: John Hawkes

The following brings up performance on a 64-way Altix.  This system being on
the smaller end of the scale should also be applicable to other NUMA systems.


Index: linux-2.6.5/include/linux/sched.h
===================================================================
--- linux-2.6.5.orig/include/linux/sched.h	2004-04-16 15:06:43.000000000 +0200
+++ linux-2.6.5/include/linux/sched.h	2004-04-16 15:06:44.000000000 +0200
@@ -620,7 +620,7 @@
 	.groups			= NULL,			\
 	.min_interval		= 8,			\
 	.max_interval		= 256*fls(num_online_cpus()),\
-	.busy_factor		= 8,			\
+	.busy_factor		= 32,			\
 	.imbalance_pct		= 125,			\
 	.cache_hot_time		= (10*1000000),		\
 	.cache_nice_tries	= 1,			\
